src/App.tsx

```diff
@@ -1,24 +1,258 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState,
+  useCallback,
+  useEffect,
+  useMemo,
+} from 'react';
+
+// components
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorMessage } from './components/ErrorMessage';
+
+// utils for loading and updating
```

Comment:
I believe  final code should be clear without comments , you can left comment like TODO or explain hard logic in extra cases , but write comment to import its overkill 😃